### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
1. Added grid component
2. Added documentation of grid component

image

### CODE REVIEW

File: components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+:root{
+    --black:black;
+    --brown:brown:brown;
```
Comment (rahulk31) [inline components/grid/grid.css]: I believe this is by mistake, you can fix this.

File: components/grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            <li><a href="/components/html/nav.html">Navigation-Desktop</a></li>
+            <li><a href="/components/html/modal.html">Modal</a></li>
+            <li><a href="/components/html/rating.html">Rating</a></li>
+            <li><a href="/components/html/toast.html">Toast/Snackbar</a></li>
+            <li><a href="/components/html/card.html">Simplified Grid</a></li>
+        </ul>
+    </div>
+
+    <div class="Component-container">
+
```
Comment (rahulk31) [inline components/grid/grid.html:49]: Can make use of sementic tag like section

File: components/grid/grid.css (hunk 1 above)
Comment (JuvitaSaini09) [reply components/grid/grid.css]: yes, i have fixed it,thnks for the review

**Review by rahulk31 — APPROVED** [review]
Classnames have been used properly 👍 Looks good to merge.

File: components/grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    font-size:30px;
+ }
+
+ .grid2-container{
+    border:2px solid var(--black);
+    padding:1rem;
+    gap:5px;
+    display:grid;
+    grid-template-columns:repeat(3,1fr);
+    width:90%;
```
Comment (gauravsinhaweb) [inline components/grid/grid.css:31]: Change width to rem

File: components/grid/grid.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+      
+         <!---------simplified-grid componenet-------->
+         <div class="text pb2 " id="grid">
+            <h2 class="pb1 h2 comp-title">Simplified grid</h2><br>
```
Comment (gauravsinhaweb) [inline components/grid/grid.html]: Remove br tag if there is no use.

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: components/grid/grid.html (hunk 4 above)
Comment (JuvitaSaini09) [reply components/grid/grid.html]: done

File: components/grid/grid.css (hunk 3 above)
Comment (JuvitaSaini09) [reply components/grid/grid.css:31]: ok,i will take care from next time

File: components/grid/grid.html (hunk 2 above)
Comment (JuvitaSaini09) [reply components/grid/grid.html:49]: i will take care if it from next time,thnku for the review

File: components/grid/grid.css (hunk 3 above)
Comment (gauravsinhaweb) [reply components/grid/grid.css:31]: okay

**Review by gauravsinhaweb — APPROVED** [review]

**Pull request merged**